source/administration/config-settings.rst

```diff
@@ -3312,6 +3312,14 @@ The email address your Global Relay server monitors for incoming compliance expo
 | This feature's ``config.json`` setting is ``"EmailAddress": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Global Relay SMTP Server Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 0 with a maximum setting of 1800 milliseconds.
```

Comment:
```suggestion
The number of milliseconds that can elapse before the connection attempt to the SMTP server is abandoned. The default value is 1800 seconds. There is no maximum value.
```

Reply:
I removed the maximum value sentence, there is no need for mentioning it :)